[ethers-v4] New factories folder causing module resolution problem.

Since [this fix](https://github.com/ethereum-ts/TypeChain/issues/225), I'm getting the following issue inside a generated factory:

```
src/generated/factories/LinkToken__factory.ts(9,38): error TS2307: Cannot find module '.'
```

It appears this line is trying to import TransactionOverrides from the factories folder:

```
import { TransactionOverrides } from ".";
```

This works if the "." becomes ".."

--- a/packages/target-ethers-v4/src/codegen/index.ts
+++ b/packages/target-ethers-v4/src/codegen/index.ts
@@ -94,7 +94,7 @@ export function codegenContractFactory(contract: Contract, abi: any, bytecode?: 
   import { UnsignedTransaction } from "ethers/utils/transaction";
   ${ethersUtilsImportLine}
 
-  import { TransactionOverrides } from ".";
+  import { TransactionOverrides } from "..";
   import { ${contract.name} } from "../${contract.name}";
 
   export class ${contract.name}${FACTORY_POSTFIX} extends ContractFactory {

